tests/model_serving/model_server/storage/pvc/utils.py

```diff
@@ -10,31 +11,64 @@ def create_isvc(
     name: str,
     namespace: str,
     deployment_mode: str,
+    storage_key: str,
     storage_uri: str,
     model_format: str,
     runtime: str,
-    min_replicas: int = 1,
+    max_replicas: Optional[int] = 1,
+    min_replicas: Optional[int] = 1,
+    cpu_limit: str = "2",
+    memory_limit: str = "8Gi",
+    cpu_request: str = "1",
+    memory_request: str = "4Gi",
```

Comment:
why do we want to set these defaults? why not have the same values set for limits and requests? what happens if you do not set defaults?